### ISSUE TITLE
[Suggestion] Olive green

### ISSUE THREAD

**Issue by ZachZent** [tsunamayo/Starship-EVO#2569]
![](https://cdn.discordapp.com/attachments/406664114249793536/726914272172703814/unknown.png)

![](https://cdn.discordapp.com/attachments/416013956331601930/726905097623437382/20200628220102_1.jpg)

Suggestion to change dark green, row 3 column 5, to 3C4830

![](https://cdn.discordapp.com/attachments/416013956331601930/726914775182737518/unknown.png)

![](https://cdn.discordapp.com/attachments/416013956331601930/726913692620292207/unknown.png)

**Comment by BrickMacklin** [comment]
I would advocate for shades of Olive Greens by themselves. As it stands now all of the dark colors have a neon look to them but green is easily the worst offender. Not that it isn't nice that we have but some proper actual dark/muted colors across the spectrum would be **MIGHTY** useful.

Regarding green specifically, here is a project I'm working on
![image](https://user-images.githubusercontent.com/37836252/85959025-accb5c00-b967-11ea-9aaa-50f5985756a0.png)

And here is what I am trying to emulate. 
![414381_391075260913297_1646372279_o](https://user-images.githubusercontent.com/37836252/85959033-b81e8780-b967-11ea-9d84-27b74c76a381.jpg)

There is nothing in the game that really matches that. Additionally there have been a recent number of people on the discord creating tanks that have been wanting an olive color. They've made us of your new tans (and they are _excellent_ thank you for them) shades of olive greens would only serve to compliment them. I do hope this can be a new addition.

And by they way a sincere thank you for adding the browns and tans. They have easily been a welcome compliment.

**Comment by MRonlin** [comment]
^I've been suck trying to finish off my halo build without that olive green. 
![711980_20200629000727_1](https://user-images.githubusercontent.com/64201628/85961845-0bfb9180-b9a5-11ea-8f42-91e79c230633.png)

**Comment by SovietSponge** [comment]
We require olive green urgently

**Comment by ExodistSKY1** [comment]
what would be really cool is if you can bring back custom colors in a more efficient way than before!

**Comment by tsunamayo** [comment]
yeah at this point of course I cant keep up with all your replica requirement, so custom colors are needed...

**Comment by ZachZent** [comment]
Absolutely, but for the time being olive green will be great for all. Not just replica makers. It is a more standard color people use and it will broaden the range of default colors we have now. Especially since there are six greens which are rather similar
